lib/modules/apostrophe-login/index.js

```diff
@@ -454,6 +537,89 @@ module.exports = {
       });
     };
 
+    // Returns an array of error messages, which will be
+    // empty if there are no errors. The error messages
+    // will be internationalized for you.
+
+    self.checkPasswordRules = function(req, password) {
+      var errors = [];
+      var minLength = self.options.passwordMinLength || 1;
+      if (password.length < minLength) {
+        errors.push('password must be at least ' + minLength + ' characters long');
+      }
+      if (self.options.passwordRules) {
+        _.each(self.options.passwordRules, function(name) {
+          var rule = self.passwordRules[name];
+          if (!rule.test(password)) {
+            errors.push(rule.message);
+          }
+        });
+      }
+      errors = errors.map(function(error) {
+        return req.__(error);
+      });
+      return errors;
+    };
+
+    self.passwordRules = {
+      noSlashes: {
+        test: function(password) {
+          return !((password.indexOf('/') !== -1) || (password.indexOf('\\') !== -1));
+        },
+        message: '/ and \\ characters are not allowed in password'
```

Comment:
I guess the assumption with these is that they would be overridden for non-English?

Reply:
No, they are passed through `__` later.